### PR TITLE
Issue 37

### PR DESCRIPTION
**Pull request by ttahm3d**
PR for issue #37 

- Removed `Noto Sans` and added `Open Sans` as font family for headings
- Using `<link>` imports to improve performance in lighthouse because of fonts
- Removed scrollbar styling on body tag moved it to class named `body`
- Changed typography added `h1` to `h6` and moved it to seperate classes `heading-1` to `heading-6`
- Added `--pure-white` and `--pure-black` css variables
- Added `.normal` utility class for text
- Renamed classes to be generic

### CODE REVIEW

File: css/components/buttons.css [hunk 1]
```diff
@@ -3,15 +3,28 @@
...
   border: 0;
   padding: 0.5rem 0.8rem;
+  transition: 0.2s all ease;
   border-radius: 0.25rem;
   font-size: 1rem;
 }
 
+.btn-outline {
+  outline: 0;
+  cursor: pointer;
```
Comment (Sameony) [inline css/components/buttons.css:13]: you have given the same property to the button element as well as the btn class. 
you have also applied it on the anchor tag as well
unless you are using it elsewhere, you can omit it.

**Review by Sameony — APPROVED** [review]
Nice structure, liked the use of descriptive classes. 
just a minor doubt, you can merge after looking it up.

File: css/components/buttons.css (hunk 1 above)
Comment (ttahm3d) [reply css/components/buttons.css:13]: I dont want user to enter `btn btn-outline btn-outline-primary`. So doing it this way

Comment (Sameony) [reply css/components/buttons.css:13]: oh, alright, thanks for explaining

**Review by nakulsharma15 — APPROVED** [review]
Looks good to me. No changes suggested!

**Pull request merged**